dataprofiler/tests/labelers/test_regex_model.py

```diff
@@ -231,12 +233,13 @@ def test_predict(self, mock_stdout):
                                     model_output['conf']):
             self.assertTrue(np.array_equal(expected, output))
 
-        # test verbose = False
-        # clear stdout
-        mock_stdout.seek(0)
-        mock_stdout.truncate(0)
-        model_output = model.predict(['hello world.'], verbose=False)
-        self.assertNotIn('Data Samples', mock_stdout.getvalue())
+        # test verbose = False doesn't make any INFO logs
+        with self.assertRaisesRegex(AssertionError,
+                                    'no logs of level INFO or higher triggered '
+                                    'on DataProfiler.labelers.regex_model'):
+            with self.assertLogs('DataProfiler.labelers.regex_model',
+                                 level='INFO'):
+                model.predict(['hello world.'], verbose=False)
```

Comment:
Interesting solve. the other option would be to assertLog, put a dummy log in and then assert it is the only log. I'd have to think on what's more appropriate. It just feels weird to make an assert knowing it should fail.